netgrasp/utils/mac.py

```diff
@@ -0,0 +1,255 @@
+from __future__ import print_function
+
+import os
+import re
+import datetime
+
+TEST_WSDATA = False
+TEST_WSDATA_FILENAME = '~/.netgrasp.mac.wsdata'
+
+DB_LOOKUP_TABLE_NAME = 'macvendor'
+DB_LOOKUP_CONFIG_SOURCE_KEY = 'maclookup_source'
+DB_LOOKUP_CONFIG_UPDATED_KEY = 'maclookup_updated'
+
+LOOKUP_SOURCES = ['local', 'remote']
+
+DEFAULT_LOOKUP_SOURCE = 'remote'
```

Comment:
So perhaps we have `['auto', 'local', 'remote']` and we default to `auto`. If someone wants to force one or the other, fine, but otherwise I think we can automatically determine what's best.